test/spoom/location_test.rb

```diff
@@ -0,0 +1,100 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+require_relative "../../lib/spoom/location.rb"
+
+require "stringio"
+
+module Spoom
+  module Sorbet
+    class LocationTest < Minitest::Test
+      include Spoom::TestHelper
+
+      def test_from_string
+        location = Location.from_string("foo.rb:1:2-3:4")
+        assert_equal("foo.rb", location.file)
+        assert_equal(1, location.start_line)
+        assert_equal(2, location.start_column)
+        assert_equal(3, location.end_line)
+        assert_equal(4, location.end_column)
+
+        assert_raises(Location::LocationError) do
+          Location.from_string("foo.rb:1:2-3")
+        end
+
+        assert_raises(Location::LocationError) do
+          Location.from_string("foo.rb:1:2")
+        end
+
+        assert_raises(Location::LocationError) do
+          Location.from_string("foo.rb:1")
+        end
+
+        assert_raises(Location::LocationError) do
+          Location.from_string("foo.rb")
+        end
+      end
+
+      def test_include
+        location1 = Location.new("foo.rb", 1, 2, 3, 4)
+        location2 = Location.new("foo.rb", 1, 2, 3, 4)
+        assert(location1.include?(location2))
+
+        location3 = Location.new("foo.rb", 1, 2, 3, 5)
```

Comment:
The test could be a little more readable if we give descriptive names to the variables, e.g. `location_with_higher_column`.

Reply:
So much better! I ended up dropping `location_with_` as it was getting too long and making it hard to read. I wonder if the variable names still make sense without?

Reply:
Just to make sure we're all on the same page, should I revert to using `location1`, `location2` naming, or keep the updated descriptive variable names?

Reply:
You can put it back to `location1` etc.